pkg/query/endpointset_test.go

```diff
@@ -269,13 +271,55 @@ func (e *testEndpoints) CloseOne(addr string) {
 	delete(e.srvs, addr)
 }
 
+func TestTruncateExtLabels(t *testing.T) {
+	const testLength = 5
+	testCases := []struct {
+		labelToTruncate string
+		expectedOutput  string
+	}{
+		{
+			labelToTruncate: "{abc}",
```

Comment:
Amazing! Let's order those cases in same way to have it well organized and readable

Reply:
done